force-app/main/default/objects/CQ_RS_SQX_Employees__c/fields/CQ_RS_SQX_Bank__c.field-meta.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>CQ_RS_SQX_Bank__c</fullName>
+    <deleteConstraint>SetNull</deleteConstraint>
+    <description>The bank where the customer has an account.</description>
+    <externalId>false</externalId>
+    <inlineHelpText>Select the customer&apos;s bank from the list.</inlineHelpText>
+    <label>Bank</label>
+    <referenceTo>CQ_RS_SQX_Bank__c</referenceTo>
+    <relationshipLabel>Employees</relationshipLabel>
+    <relationshipName>Employees</relationshipName>
```

Comment:
we will be also following the naming convention in relationship name too

Reply:
done sir